runtime/lua/health.lua

```diff
@@ -1,23 +1,6 @@
-local M = {}
-
-function M.report_start(msg)
-  vim.fn['health#report_start'](msg)
-end
-
-function M.report_info(msg)
-  vim.fn['health#report_info'](msg)
-end
-
-function M.report_ok(msg)
-  vim.fn['health#report_ok'](msg)
-end
-
-function M.report_warn(msg, ...)
-  vim.fn['health#report_warn'](msg, ...)
-end
-
-function M.report_error(msg, ...)
-  vim.fn['health#report_error'](msg, ...)
-end
-
-return M
+return setmetatable({}, {
+  __index = function(_, k)
+    vim.deprecate("require('health')", 'vim.health', '0.9', false)
```

Comment:
Unsure about this. Maybe we don't want users trying to see what's wrong with their installation `:checkhealth` to have a deprecation warning, or maybe it's alright since they are using a nightly version?

Reply:
Let's try it this way.